src/macad_gym/envs/follow_leading_vehicle.py

```diff
@@ -0,0 +1,139 @@
+"""
+Author: Morphlng
+Date: 2023-01-28 18:33:26
+LastEditTime: 2023-01-28 18:33:28
+LastEditors: Morphlng
+Description: Example of creating a custom environment, also a demonstration of how to use the manual_control.
+FilePath: \src\macad_gym\envs\follow_leading_vehicle.py
+"""
```

Comment:
This file comment header style is not consistent within this repository and most of this information is automatically recorded in git and displayed on GitHub. The description is good to keep as it provides a file-level description.
You can keep your name as `__author__: Morphlng` if you would like.